dist/overflow-color.umd.js

```diff
@@ -109,7 +101,13 @@
     if (bodyComputedBackground === '' || bodyComputedStyle.getPropertyValue('background-color') === 'rgba(0, 0, 0, 0)' && bodyComputedBackground.substring(21, 17) === 'none') {
       bodyComputedBackground = 'white';
     }
-    document.body.style.background = 'transparent';
+
+    var scrollHeight = document.body.scrollHeight;
+    var innerHeight = window.innerHeight;
+
+    if (scrollHeight === innerHeight) {
```

Comment:
This made just for reliability. We set background-color to `bottomColor` while scrolling down, and when we don't scroll or there is no scroll at all, we apply `topColor` so that each color in ends up in the right place.